aws/src/main/java/org/apache/iceberg/aws/AwsClientFactories.java

```diff
@@ -193,6 +193,14 @@ public static <T extends SdkClientBuilder> void configureEndpoint(T builder, Str
     }
   }
 
+  /**
+   * Build an S3Configuration object from two boolean flags, pathStyleAccess and
```

Comment:
> Build an S3Configuration object from two boolean flags, pathStyleAccess and s3UseArnRegionEnabled

This part is a bit too specific, since we are just deprecating it, we can just have a simple message like "Build an S3 configuration object". The parameters should be put in the `@param` lines

Reply:
Make sense

Reply:
Updated